config.yaml

```diff
@@ -0,0 +1,19 @@
+AMQP_URI: 'amqp://guest:guest@localhost'
+WEB_SERVER_ADDRESS: '0.0.0.0:8000'
+rpc_exchange: 'nameko-rpc'
+max_workers: 10
+parent_calls_tracked: 10
+REDIS_URI: 'redis://localhost:6379/5'
```

Comment:
Yes and no ;-) I put the _real_ default value in the config file (cf other comment about what is the config file now).